sonar-kotlin-plugin/src/main/java/org/sonarsource/kotlin/checks/VariableAndParameterNameCheck.kt

```diff
@@ -77,7 +77,8 @@ class VariableAndParameterNameCheck : AbstractCheck() {
         if (!name.matches(formatRegex)) {
             kotlinFileContext.reportIssue(
                 declaration.nameIdentifier!!,
-                "Rename this $kind to match the regular expression \"$format\".")
+                """Rename this $kind to match the regular expression "$format"."""
```

Comment:
Remove quotes & dot from regex. (And if you decide to put/leave the colons on the other rules, put one here as well).